lib/node_modules/@stdlib/math/base/special/heaviside/lib/native.js

```diff
@@ -0,0 +1,64 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var addon = require( './../src/addon.node' );
+var str2enum = require( './str2enum.js' );
+
+
+// MAIN //
+
+/**
+* Evaluates the Heaviside function.
+*
+* @private
+* @param {number} x - input value
+* @param {string} [continuity] - continuity option
```

Comment:
Should we mark the parameter as mandatory here?

```suggestion
* @param {string} continuity - continuity option
```

Reply:
The JS implementation does have it as optional, too, so probably better to leave as-is for now. There is an argument to be made to make the parameter mandatory overall in this package as we have moved to avoid variadic functions for performance reasons, but that could be handled in a separate PR.

cc @kgryte

Reply:
I suggest going ahead and making mandatory across the package.

Reply:
Cool! Updating the PR accordingly....

Thanks @Planeshifter and @kgryte for the feedback!